Dockerfile

```diff
@@ -1,11 +1,12 @@
 ARG VARIANT=2.6
+
 FROM mcr.microsoft.com/vscode/devcontainers/ruby:${VARIANT}
 
 # Install node
-COPY .nvmrc /tmp/.nvmrc
+ARG NODE_VERSION="12.1"
 RUN \
   su vscode -c \
-    "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /tmp/.nvmrc) 2>&1"
+    "source /usr/local/share/nvm/nvm.sh && nvm install $NODE_VERSION 2>&1"
```

Comment:
This should work, it could be `/tmp` that causes troubles we could try using any other path 🤷

Reply:
I have used root as path and get a different error.
```
$ docker build -t homeassistant-io .              [+] Building 1.1s (8/11)                                                                                                 => [internal] load build definition from Dockerfile                                                               0.1s  => => transferring dockerfile: 38B                                                                                0.1s  => [internal] load .dockerignore                                                                                  0.1s  => => transferring context: 2B                                                                                    0.1s  => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/ruby:2.6                                   0.1s  => [internal] load build context                                                                                  0.1s  => => transferring context: 59B                                                                                   0.1s  => [1/7] FROM mcr.microsoft.com/vscode/devcontainers/ruby:2.6@sha256:cd9e6982dec84e5f8c6d3177d30083ed5d184ab608c  0.0s  => CACHED [2/7] COPY .nvmrc /.nvmrc                                                                               0.0s  => CACHED [3/7] RUN cat /.nvmrc                                                                                   0.0s  => ERROR [4/7] RUN   su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1"     0.7s ------                                                                                                                   > [4/7] RUN   su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1":                #8 0.618 N/A: version "12.1 -> N/A" is not yet installed.                                                               #8 0.618                                                                                                                #8 0.618 You need to run "nvm install 12.1" to install it before using it.                                              ------                                                                                                                  executor failed running [/bin/sh -c su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1"]: exit code: 3
```
If I just replace $(cat /.nvmrc) by 12.1 the build runs ok. Any Idea?

Reply:
After changing to a path that works.
```
$ docker build -t homeassistant-io .
[+] Building 1.1s (8/11)
 => [internal] load build definition from Dockerfile                                                               0.1s
 => => transferring dockerfile: 38B                                                                                0.1s
 => [internal] load .dockerignore                                                                                  0.1s
 => => transferring context: 2B                                                                                    0.1s
 => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/ruby:2.6                                   0.1s
 => [internal] load build context                                                                                  0.1s
 => => transferring context: 59B                                                                                   0.1s
 => [1/7] FROM mcr.microsoft.com/vscode/devcontainers/ruby:2.6@sha256:cd9e6982dec84e5f8c6d3177d30083ed5d184ab608c  0.0s
 => CACHED [2/7] COPY .nvmrc /.nvmrc                                                                               0.0s
 => CACHED [3/7] RUN cat /.nvmrc                                                                                   0.0s
 => ERROR [4/7] RUN   su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1"     0.7s
------
 > [4/7] RUN   su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1":
#8 0.618 N/A: version "12.1 -> N/A" is not yet installed.
#8 0.618
#8 0.618 You need to run "nvm install 12.1" to install it before using it.
------
executor failed running [/bin/sh -c su vscode -c     "source /usr/local/share/nvm/nvm.sh && nvm install $(cat /.nvmrc) 2>&1"]: exit code: 3
```